### DISCUSSION TITLE
Welcome to Fliq Discussions!

### DISCUSSION THREAD

**Discussion by JahidHasanCO** [JahidHasanCO/Fliq.-The-Movie-DB#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Knowledgekamuteku** [comment, marked as answer]
Hello this is an interesting project and with your approval I would like to contribute. My name is knowledge

**Reply by JahidHasanCO** [reply]
Hi knowledge,
Thanks for replying. I am glad to see that you like this project. You can contribute this project.
